.rubocop.yml

```diff
@@ -7,3 +7,5 @@ Style/FrozenStringLiteralComment:
 Lint/AmbiguousRegexpLiteral:
   Exclude:
     - '**/test/**/*'
+Layout/EndOfLine:
```

Comment:
This was unique to my testing system and may not be desired to be included in the final PR.

Reply:
@webframp thoughts on using a `.gitattributes` file to maintain consistency here? We can force them on commit to conform to one or the other.

Reply:
Can't recall answering this in Slack or elsewhere. Definitely a fan of being explicit with `.gitattributes`